assets/src/components/v2/elevator/models/station.ts

```diff
@@ -0,0 +1,9 @@
+import { type Pill } from "Components/v2/departures/route_pill";
+import { type Closure } from "Components/v2/elevator/models/closure";
+
+export type StationWithClosures = {
```

Comment:
I think these could both live in a `v2/elevator/types.ts`. Splitting them out at this point feels premature, and I don't love the name "model" when these don't have anything to them other than the type definition (it kinda implies some larger framework involving "models" as object-oriented entities that can be persisted or cached or etc.)